<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1497</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectra databases</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btFirst">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-first.svg</normaloff>:/icons/images/go-first.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btPrevious">
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-previous.svg</normaloff>:/icons/images/go-previous.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblOffset">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btNext">
       <property name="text">
        <string>&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-next.svg</normaloff>:/icons/images/go-next.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btLast">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-last.svg</normaloff>:/icons/images/go-last.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editGoTo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btGoTo">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/go-jump.svg</normaloff>:/icons/images/go-jump.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBanks">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btRefresh">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/refresh.svg</normaloff>:/icons/images/refresh.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tvSpectra">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>0</height>
       </size>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="SpectrumDetailsWidget" name="widgetSpectrumDetails" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpectrumDetailsWidget</class>
   <extends>QWidget</extends>
   <header>lib/ui/widgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btFirst</tabstop>
  <tabstop>btPrevious</tabstop>
  <tabstop>btNext</tabstop>
  <tabstop>btLast</tabstop>
  <tabstop>editGoTo</tabstop>
  <tabstop>btGoTo</tabstop>
  <tabstop>cbBanks</tabstop>
  <tabstop>btRefresh</tabstop>
  <tabstop>tvSpectra</tabstop>
 </tabstops>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
